docs/zkapps/how-zkapps-work.mdx

```diff
@@ -98,11 +101,11 @@ When you deploy to a new Mina address, the Mina Protocol will charge a 1 MINA fe
 <img src="/img/7_zkApps_DeploySmartContract.jpg" width="95%" />
 <br></br>
 
-Developers usually build a UI to allow users to interact with the smart contract.
+zkApp developers usually build a UI to allow users to interact with the smart contract.
 
-Typically, this is a static website deployed to a host of the developer’s choosing. We recommend web hosts that offer a global CDN to ensure the best user experience.
+Typically, this static website is deployed to a host of the developer’s choosing.  Recommend web hosts offer a global CDN to ensure the best user experience.
 
-Your website needs to contain the `smart_contract.js` file that was generated when building your smart contract. On the [how to write a zkApp](how-to-write-a-zkapp) page, we will go into more detail.
+Your website must contain the `smart_contract.js` file that was generated when building your smart contract. See [how to write a zkApp](how-to-write-a-zkapp) page for more detail.
```

Comment:
@ymekuria This is out of date isn't it? Should this file name be updated?

Reply:
Would wait to get confirmation on name of `smart_contract.js` before merging. It's important

Reply:
This is outdated. The sentence is was trying to say that a smart contract must be included in the UI in some way.

Reply:
Thanks Yoni.

@barriebyron In other words, the output file name is based on their smart contract class name: i.e. `YourSmartContract.js' and isn't a fixed name.

Suggestion:
> Your website must contain the JavaScript version of your smart contract that is output when building your TypeScript smart contract project. 

Maybe someone can think of something better, but that seems decent.